therapy/schemas.py

```diff
@@ -347,10 +406,12 @@ def schema_extra(schema: Dict[str, Any],
             }
 
 
-class FDAStatus(BaseModel):
-    """VOD Extension class for FDA status/indication attributes."""
+class ApprovalStatusValue(BaseModel):
+    """VOD Extension class for regulatory approval status/indication
+    value attributes.
+    """
 
-    approval_status: Optional[ApprovalStatus]
+    approval_status: Optional[List[ApprovalStatus]]
     approval_year: Optional[List[int]]
     has_indication: Optional[List[Union[Dict, ValueObjectDescriptor]]]
```

Comment:
Should we pluralize these fields since they are lists?

Reply:
tbh I'd really like to come up with a better word than "status" because it's really more of a source-by-source "evaluation" or "rating" rather than "status". To me it seems awkward to have multiple, potentially conflicting "statuses". To the thesaurus!

Reply:
"approval status" is typically how this information referred. I think pluralizing this field will lead to confusion, especially since there can be disagreement between sources. 

I totally agree, though, after working through all the different ways sources represent this information, that each of these sometimes feel like more of an 'evaluation' or 'rating' than an explicitly agreed upon 'status' (maybe even just an 'approval_field'?)

Reply:
`approval_status` --> `approval_ratings`